### ISSUE TITLE
Feature Request: Logo on Notification Emails.

### ISSUE THREAD

**Issue by ericdude101** [snipe/snipe-it#3874]
#### Expected Behavior (or desired behavior if a feature request)

I would like the ability to embed our company logo to the notification emails. 

-----

#### Actual Behavior

Currently as a band-aid, I have been able to insert an external image into the email template but this gets blocked by most email clients for security reasons. I tried embedding the image using base64 but it didn't seem to work.

-----
#### Please provide answers to these questions before posting your bug report:

- Version of Snipe-IT you're running: **3.6.6 build 1**
- What OS and web server you're running Snipe-IT on: **Server 2012R2 IIS8**
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc): **Install.sh**
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error: **Email Templates**
- Indicate whether or not you've manually edited any data directly in the database: **No**

**Comment by snipe** [comment]
Closing as duplicate of #3831
